src/index.js

```diff
@@ -142,10 +153,32 @@ const App = () => {
                     <ToDoItems todos={sortedTodos} onCompleteItem={handleCompleteItem} />
                 </tbody>  
             </table>
-            <input type='text' placeholder='event name' value={todoInputValue} onChange={e => setTodoInputValue(e.target.value)}  />
-            <input type="date" min={tomrISO} value={dateInputValue} onChange={e => setDateInputValue(e.target.value)} /> <button onClick={handleAddItem} disabled={isAddButtonDisabled}> Add </button>
+
+            <input 
+                className={!todoInputValue && hasTaskInputBeenTouched ? 'invalid' : null}
+                type='text' 
+                placeholder='event name'
+                value={todoInputValue} 
+                onBlur={() => setHasTaskInputBeenTouched(true)}
+                onChange={e => setTodoInputValue(e.target.value)}
+            />
+            <input 
+                className={isDateInvalid ? 'invalid' : null}
+                type="date" min={tomrISO} 
+                value={dateInputValue}
+                onChange={e => validateDueDate(e.target.value)}  
+            />
+            <button 
+                onClick={handleAddItem} 
+                disabled={isAddButtonDisabled}> 
+                Add 
+            </button>
```

Comment:
This chunk is mostly formatting enhancements.
And deciding whether to put a red border on the 2 input fields using `setHasTaskInputBeenTouched` and `isDateInvalid`